astropy/io/fits/tests/test_table.py

```diff
@@ -1049,14 +1049,14 @@ def test_assign_multiple_rows_to_table(self):
         assert tbhdu2.columns.columns[2].array[4] == ''
         assert (tbhdu2.columns.columns[3].array[4] ==
                 np.array([1., 2., 3., 4., 5.], dtype=np.float32)).all()
-        assert tbhdu2.columns.columns[4].array[4] == False
+        assert tbhdu2.columns.columns[4].array[4] is False
```

Comment:
TIL about the diff between ``bool`` and ``numpy.bool_``. So the question is whether we want to:
 a) enforce E712, or 
 b) skip the few lines with pylint where both ``bool`` and ``numpy.bool_`` can be used, or
 c) convert one of them to the other

I would go with c) and have this here: ``assert bool(tbhdu2.columns.columns[4].array[4]) is False``